From c1e3014597d3662cc45259d6b6ec69c5e7e1f214 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sat, 5 Jun 2021 17:16:27 +0200
Subject: [PATCH] adapt python requirement for py38 on OSX-ARM

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index fd16e7b..ed6891e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,7 +3,7 @@ requires = [
     "numpy>=1.17,<1.18; python_version=='3.5'",
     "numpy>=1.17,<1.18; python_version=='3.6'",
     "numpy>=1.17,<1.18; python_version=='3.7'",
-    "numpy>=1.17,<1.18; python_version=='3.8'",
+    "numpy>=1.19,<1.20; python_version=='3.8'",
     "numpy>=1.19,<1.20; python_version=='3.9'",
     "setuptools",
     "wheel",
-- 
2.31.1.windows.1

